pkg/network/protocols/http2/kernel_requirements.go

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package http2
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/util/kernel"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// MinimumKernelVersion indicates the minimum kernel version required for HTTP2 monitoring
+var MinimumKernelVersion kernel.Version
+
+func init() {
+	MinimumKernelVersion = kernel.VersionCode(5, 2, 0)
+}
+
+// Supported We only support http2 with kernel >= 5.2.0.
```

Comment:
can you add here the reason why http2 supported only from 5.2.0? (is it an instructions limit?)

Reply:
done